internal/sql/repository/NotificationSettingsRepository.go

```diff
@@ -17,12 +17,17 @@
 package repository
 
 import (
+	"fmt"
 	"github.com/devtron-labs/devtron/pkg/sql"
 	"github.com/go-pg/pg"
 	"github.com/go-pg/pg/orm"
+	"k8s.io/utils/pointer"
 	"strconv"
 )
 
+const AllExistingAndFutureNonProdEnvsInt = -1
```

Comment:
can we move it to resource qualifier package to keep it consistent with ent

Reply:
done